packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.js

```diff
@@ -1736,7 +1745,8 @@ describe('src/cy/commands/actions/type - #type special chars', { browser: '!webk
         })
       })
 
-      it('will not submit the form', function (done) {
+      // WebKit still emits the submit event on the form in this configuration.
```

Comment:
Is this just how WebKit works? It acts the same if a user presses "Enter" on a form with multiple submits?

Reply:
Correct, I recreated this scenario outside of Cypress and found the form was still submitted in this case in WebKit, whereas its not in Chrome/FF.